tests/water_fugacity_prefactor.prm

```diff
@@ -0,0 +1,123 @@
+# In the solid_2 composition, r=0.7 and the viscosity should be equal to 4.924e23
+# Which should result in an average model viscosity of 2.462e23.
+# The material model should return this viscosity.
+
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 1
+set Surface pressure                       = 10.e9
+
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = solid_2, bound_fluid
+end
+
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Function expression = if(y<=50e3, 1.0, 0.0); 0.01
+  end
+end
+
+# Model geometry (100x100 km, 10 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# No boundary-driven deformation, but the strain-rate used by the material
+# model is set in the material model section through the parameters
+# "Reference strain rate" and "Minimum strain rate".
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, top, left, right
+end
+
+# Initial temperature conditions
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 1673
+  end
+end
+
+# Material model (values for background material)
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+    # dislocation creep parameters
+    # These values produce such a sufficiently high viscosity that the
+    # composite viscosity between the diffusion and dislocation creep
+    # is effectively equal to the Peierls creep viscosity.
+    set Viscous flow law                          = diffusion
+    set Prefactors for diffusion creep            = 1.57739e-21, 2.5e-17, 2.5e-17
+    set Stress exponents for diffusion creep      = 1.0
+    set Activation energies for diffusion creep   = 375e3
+    set Activation volumes for diffusion creep    = 15e-6
+    set Grain size                                = 1e-3
+    set Exponents for water fugacity              = 0.7, 0.0, 0.0
+    set Viscosity prefactor scheme                = water fugacity
+  end
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 0.0
+  end
+end
+
+# Post processing
+subsection Postprocess
+  set List of postprocessors = material statistics
```

Comment:
I would add visualization as one of the postprocessors, and then output in gnu format to confirm the switch in viscosity across 50 km depth.